mod_reforged/hooks/config/strings.nut

```diff
@@ -447,7 +447,7 @@ local vanillaDescriptions = [
 			Effects = [{
 				Type = ::UPD.EffectType.Passive,
 				Description = [
-					"With no adjacent ally of your faction, gain a " + ::MSU.Text.colorPositive("+15%") + " bonus to [Melee Skill,|Concept.MeleeSkill] [Ranged Skill,|Concept.RangeSkill] [Melee Defense,|Concept.MeleeDefense] [Ranged Defense,|Concept.RangeDefense] and [Resolve.|Concept.Bravery]"
+					"When no ally of your faction is adjacent to you, gain a " + ::MSU.Text.colorPositive("+15%") + " bonus to [Melee Skill,|Concept.MeleeSkill] [Ranged Skill,|Concept.RangeSkill] [Melee Defense,|Concept.MeleeDefense] [Ranged Defense,|Concept.RangeDefense] and [Resolve.|Concept.Bravery]"
```

Comment:
Take this opportunity to change the `+15%` to `15%` here to show that its multiplicative.